SayTheirNames/Source/Controller/Donations/DonationsController.swift

```diff
@@ -63,6 +76,25 @@ final class DonationsController: UIViewController {
             return cell
         }
 
+        donationManager.didSelectItem = { donation in
```

Comment:
Missing weak self, and in couple other closures too (there’s a retain cycle)

Reply:
Okay, thanks, I will implement the change.